procedures/unit-testing-junit.ipf

```diff
@@ -166,119 +73,129 @@ static Function/S JU_TrimSOUT(input, [listSepStr])
 	return output
 End
 
-/// Returns the current TimeStamp in the form yyyy-mm-ddThh:mm:ss in local time
-static Function/S JU_GetISO8601TimeStamp()
+/// Formats the specified TimeStamp in the form yyyy-mm-ddThh:mm:ss in local time
+static Function/S GetISO8601TimeStamp(localtime)
```

Comment:
It went to nirvana. I don't think a static function need a prefix because it is only accessible inside the procedure file.

Reply:
@Garados007 

> It went to nirvana. I don't think a static function need a prefix because it is only accessible inside the procedure file.

Function prefixes are always present, not matter if the function is static or not. In case this is missing from the coding conventions please create an issue there.